pyart/retrieve/gate_id.py

```diff
@@ -58,14 +58,18 @@ def map_profile_to_gates(profile, heights, radar, toa=None,
     rg, azg = np.meshgrid(radar.range['data'], radar.azimuth['data'])
     rg, eleg = np.meshgrid(radar.range['data'], radar.elevation['data'])
     _, _, z = antenna_to_cartesian(rg / 1000.0, azg, eleg)
+    
+    # Check that z is not a MaskedArray
+    if type(z) is np.ma.core.MaskedArray:
+        z = z.filled(np.NaN)
 
     # find toa is not provided
     if toa is None:
         ismasked = np.where(np.ma.getmaskarray(profile))[0]
         if len(ismasked) == 0:
             toa = None
         else:
-            toa = ismasked.min()
+            toa = ismasked.min()            
```

Comment:
@vlouf The fix looks good to me! I would just suggest removing the whitespace after this line to be pep8 :)